tests/test_centralerekenapplicatie_ingress.py

```diff
@@ -0,0 +1,258 @@
+import json
+import logging
+
+import pytz
+from django.conf import settings
+from django.test import override_settings
+from rest_framework.test import APITestCase
+
+from centralerekenapplicatie_v1.ingress_parser import CraParser
+from centralerekenapplicatie_v1.models import CraCount, CraModel
+from ingress.models import Endpoint, IngressQueue
+from peoplemeasurement.models import Sensors
+
+log = logging.getLogger(__name__)
+timezone = pytz.timezone("UTC")
+
+AUTHORIZATION_HEADER = {'HTTP_AUTHORIZATION': f"Token {settings.AUTHORIZATION_TOKEN}"}
+
+
+TEST_POST_AREA = """
+{
+    "id": "Zone 0",
+    "area": 74.6,
+    "count": 1,
+    "density": 0.013405,
+    "totalDistance": 4.2758,
+    "totalTime": 3.1,
+    "speed": 1.3793,
+    "source": {
+        "sensor": "GABM-03",
+        "timestamp": "2021-01-18T00:28:00Z",
+        "originalId": 147751,
+        "adminId": 1
+    },
+    "type": "areaMetrics"
+}
+"""
+
+TEST_POST_LINE = """
+{
+    "id": "Line 0",
+    "counts": [{
+        "azimuth": 50.0,
+        "count": 1
+    }, {
+        "azimuth": 230.0,
+        "count": 0
+    }],
+    "source": {
+        "sensor": "GACM-04",
+        "timestamp": "2021-01-18T00:28:00Z",
+        "originalId": 164240,
+        "adminId": 1
+    },
+    "type": "lineMetrics"
+}
+"""
+
+
+class DataIngressPosterTest(APITestCase):
```

Comment:
In bijna alle testcases doe je bijna meer een integration test dan dat je echte units test. Vaak maak je de data aan via de view, en vervolgens test je de parser. 

Met unit tests wil je liever meer focus; test de view, en enkel dat, of test de parser, en enkel dat. In het laatste geval zou je de data dan zelf eerst maken, maar niet via de view. 

Hoeft nu geen aanpassing, meer een tip voor volgende keer.